pages/common/xplr.md

```diff
@@ -0,0 +1,24 @@
+# xplr
+
+> Explore the file system on your terminal.
+> More information: <https://github.com/sayanarijit/xplr>.
+
+- Open a directory:
+
+`xplr {{path/to/directory}}`
+
+- Focus on a file:
+
+`xplr {{path/to/file}}`
+
+- Edit file(s):
+
+`vim $(xplr)`
+
+- Copy file(s):
+
+`cp -var $(xplr) "$(xplr)/"`
+
+- Search and move file(s):
+
+`mv -v $(fzf -m) "$(xplr)/"`
```

Comment:
I'm not _totally_ sure about this one because it introduces **2**, not 1 other command, but I doubt that really matters.

Reply:
We could base on this comment https://github.com/tldr-pages/tldr/pull/5449#issuecomment-808467937

Reply:
So `xplr` as of now doesn't have much command-line functionality. So I think I can remove the `mv` example.
Shall I also remove the `cp` and `vim` examples?

Reply:
@sayanarijit just wait for another review.

Reply:
I'd prefer it if we could remove this example. It's demonstrating the functionality of other commands rather than that of the command in question.